guides/05_tabular-data-science-and-plots/creating-a-dashboard-from-supabase-data.md

```diff
@@ -0,0 +1,122 @@
+# Creating a Dashboard from Supabase Data
+
+Tags: TABULAR, DASHBOARD, PLOTS 
+
+Supabase is a cloud-based open-source backend that provides a PostgreSQL database, authentication, and other useful features for building web and mobile applications. In this tutorial, we will show you how to read data from Supabase plot it in **real-time** on a Gradio Dashboard.
+
+**Prerequisites:** To start, you will need to have a free Supabase account, which you can sign up for here: https://app.supabase.com/
+
+We'll cover the following steps in this end-to-end Guide:
+
+* Creating tables in Supabase
+* Writing data to Supabase using the Supabase Python Client
+* Visualize the data in a real-time dashboard using Gradio
+
+If you already have data on Supabase that you'd like to visualize in a dashboard, you can skip sections and first two sections and go directly to [visualizing the data]()!
+
+## Creating a table in Supabase
+
+First of all, we need some data to visualize. Following this [excellent guide](https://supabase.com/blog/loading-data-supabase-python), we'll create some fake commerce data and put it in Supabase. 
+
+1. We'll start by creating a new project in Supabase. Once you're logged in, click the "New Project" button
```

Comment:
There's a numbering issue when the markdown gets rendered in html. You have to indent the sections in between. 

<img width="994" alt="Screen Shot 2023-03-01 at 12 11 43 AM" src="https://user-images.githubusercontent.com/9021060/222082090-373791ad-04f6-49f1-b21f-16e9b3672d5f.png">


<img width="1001" alt="Screen Shot 2023-03-01 at 12 11 29 AM" src="https://user-images.githubusercontent.com/9021060/222081861-6e6335fe-ecf6-4574-907e-e32308d9d59c.png">

To fix the above 
```
...table called `Product`, with the following schema:

<center>
<table>
<tr><td>product_id</td><td>int8</td></tr>
<tr><td>inventory_count</td><td>int8</td></tr>
<tr><td>price</td><td>float8</td></tr>
<tr><td>product_name</td><td>varchar</td></tr>
</table>
</center>

5. Click Save to save...
``` 

should be 

```
...table called `Product`, with the following schema:

    <center>
    <table>
    <tr><td>product_id</td><td>int8</td></tr>
    <tr><td>inventory_count</td><td>int8</td></tr>
    <tr><td>price</td><td>float8</td></tr>
    <tr><td>product_name</td><td>varchar</td></tr>
    </table>
    </center>
    
5. Click Save to save...
```

And same for the code.  


But I see you want to connect all the numbers together in the guide, instead of how it's rendering now, which restarts the count in every section

<img width="667" alt="Screen Shot 2023-03-01 at 12 20 05 AM" src="https://user-images.githubusercontent.com/9021060/222083192-2cdd96f2-cf9d-44ac-9616-384d5d279439.png">

If you actually wanna connect the numbers so it's 1 -> 10 just add a backslash before the period in every number: 

```
1\. 
2\.
3\.
```
You don't need indent anymore in this case.